build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     implementation "javax.inject:javax.inject:1"
     implementation "org.springframework.boot:spring-boot-starter-actuator:${springBootVersion}"
     implementation "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
-    implementation "org.springframework.data:spring-data-solr:4.1.2.RELEASE"
+    implementation "org.springframework.data:spring-data-solr:4.1.3.RELEASE"
```

Comment:
We shouldn't need solr in store. I'll work on a PR to remove it.

Reply:
https://github.com/connexta/ion-store/pull/249

Reply:
This PR should automatically close once https://github.com/connexta/ion-store/pull/249 is merged.